modules/@angular/router/src/router_module.ts

```
@@ -194,6 +192,7 @@ export function provideForRootGuard(router: Router): any {
  * @stable
  */
 export function provideRoutes(routes: Routes): any {
+  validateConfig(routes);
```

I just wanted to reuse already existing functionality and fail fast with "descriptive" error msg